web-service-consumer/1.6/modules/ROOT/pages/web-service-consumer-troubleshooting.adoc

```diff
@@ -0,0 +1,166 @@
+
+== Enable Wire Logging
+
+To begin troubleshooting Web Service Consumer Connector, look at the transport request and response on the Web Service Consumer.
+If you are using the underlying HTTP transport, you can enable wire logging to see the exact HTTP message, the transmission headers, and the response status.
+
+To enable wire logging:
+
+. Access Anypoint Studio and navigate to the *Package Explorer* view.
+. Open your application's project name.
+. Open the *src/main/resources* path folder.
+. Open the *log4j2.xml* file inside the folder.
+. If the following line is already in the *log4j2.xml* file, uncomment it to enable it; otherwise, add it:
++
+[source,xml,linenums]
+----
+<AsyncLogger name="org.mule.service.http.impl.service.HttpMessageLogger" level="DEBUG" />
+----
++
+. Save your changes.
+. Click the project name in *Package Explorer* and then click *Run* > *Run As* > *Mule Application*.
+. Navigate to the *Console* view to read the HTTP messages in the logger:
++
+
+[source,plain-text]
+----
+POST /essentials/delta HTTP/1.1
+soapaction:
+Host: ilt.mulesoft-training.com
+User-Agent: AHC/1.0
+Connection: keep-alive
+Accept: */*
+Content-Type: text/xml; charset=UTF-8
+Transfer-Encoding: chunked
+
+b1
+<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/"><soap:Body><ns0:listAllFlights xmlns:ns0="http://soap.training.mulesoft.com/"/></soap:Body></soap:Envelope>
+
+DEBUG 2021-04-06 15:04:23,058 [[wsc-base].wsc-dispatcher.11 SelectorRunner] [processor: ; event: ] org.mule.service.http.impl.service.HttpMessageLogger.wsc-dispatcher: REQUESTER
+0
+
+
+DEBUG 2021-04-06 15:04:23,319 [[wsc-base].wsc-dispatcher.11 SelectorRunner] [processor: ; event: ] org.mule.service.http.impl.service.HttpMessageLogger.wsc-dispatcher: REQUESTER
+HTTP/1.1 200
+Date: Tue, 06 Apr 2021 18:04:23 GMT
+Content-Type: text/xml; charset=UTF-8
+Transfer-Encoding: chunked
+Connection: keep-alive
+MULE_ENCODING: UTF-8
+Strict-Transport-Security: max-age=31536000; includeSubdomains;
+X-Content-Type-Options: 1
+X-Frame-Options: SAMEORIGIN
+X-XSS-Protection: 1; mode=block
+
+bfe
+<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/"><soap:Body><ns2:listAllFlightsResponse xmlns:ns2="http://soap.training.mulesoft.com/">...
+----
+
+== Troubleshoot Access Attachments
+
+Some servers respond to requests by sending MTOM (Message Transmission Optimization Mechanism) messages. These messages separate their data into parts, which enables different types of content data to coexist in the same HTTP message.
+
+MTOM is in charge of sending binary data in a separate part and referencing that part from the corresponding field within the SOAP envelope.
+Web Service Consumer users interact with these messages (multipart) only with the MTOM configuration. This is not to say that MTOM, XOP, and MIME or multipart are the same.
```

Comment:
Do you need to say that these things are not the same? I am not sure why you need to say this.

Maybe "Web Service Consumer users use the MTOM configuration to interact with these messages. However, MTOM, XOP, and MIME or multipart are not the same.

Is multipart part of MIME? If not "...MIME, and multipart are not the same."